internal/client/grpc/things/thingsclient.go

```diff
@@ -0,0 +1,40 @@
+package things
+
+import (
+	"io"
+	"io/ioutil"
+
+	"github.com/mainflux/mainflux/pkg/errors"
+	gogrpc "google.golang.org/grpc"
+
+	"github.com/mainflux/mainflux"
+	thingsapi "github.com/mainflux/mainflux/things/api/auth/grpc"
+
+	grpcClient "github.com/mainflux/mainflux/internal/client/grpc"
+
+	"github.com/mainflux/mainflux/internal/env"
+)
+
+const envThingsAuthGrpcPrefix = "MF_THINGS_AUTH_GRPC_"
+
+var (
+	errGrpcConfig = errors.New("failed to load grpc configuration")
+)
```

Comment:
Inline.

Reply:
Please don't manually resolve if the comment is not addressed.

Reply:
Sorry,  I thought it was fixed and resolved. 
But It is not actually fixed.